src/oops/SOLID/singleResponsibilityPrinciple/before/SaveEmployeesMain.java

```diff
@@ -10,7 +16,12 @@ public static void main(String[] args) {
 
         // Save all
         for (Employee e : employees){
-            Employee.save(e);
+            try {
+				repository.save(e);
+			} catch (IOException e1) {
+				ConsoleLogger.error("Exception caught while saving employee data");
+				throw new EmployeePersistException(e1);
```

Comment:
This will change the behavior as you are stopping the process when you receive the first exception. In the original code, the behavior is that it should log the exception and then proceed with saving other employee data.

Reply:
Got it! I'll include the exception stack trace in the logger error message.